### ISSUE TITLE
XP drops

### ISSUE THREAD

**Issue by TimGoll** [McJty/RFTools#1467]
The Building Block in quarry mode dropx massive amounts of XP. This is very annoying, bacause it slows my client down until it reaches 1fps.
Is there a config option to disable this? Or can you add one?

Minecraft 1.12.2, newest forge, newest RF tools

**Comment by McJty** [comment]
Hmm, pretty hard... Not sure how I would go about doing that. Have to think about it

**Comment by TimGoll** [comment]
Oh, I thought you were dropping the XP, because they drop out of your machine

maybe add an internal XP buffer (mob_grinding_utils can store XP for example). But your internal XP buffer can be voided

**Comment by McJty** [comment]
No, the drops are from the blocks that are being harvested and the problem is that when I harvest a block these XP orbs are generated out of my control

**Comment by TimGoll** [comment]
A workaround would be to destroy all `EntityXPOrb` in a certain area
